reason: Stack didn't pass verifier at the start of function
from StackPush at source/stack.cpp:133
ERROR 5: Size exceeded capacity
stack: stack1 [00000001231FFAE0]
{
	size = 36;
	capacity = 16;
	data [000001CDB5A79CB0];
	{
		*[0] = 10;
		*[1] = 20;
		*[2] = 30;
		*[3] = 40;
		*[4] = 50;
		*[5] = 60;
		*[6] = 70;
		*[7] = 80;
		*[8] = 90;
		*[9] = 89;
		*[10] = 79;
		*[11] = 69;
		*[12] = 59;
		 [13] = -558908742 (POISON);
		 [14] = -558908742 (POISON);
		 [15] = -558908742 (POISON);
	}
}